### DISCUSSION TITLE
HEXO + Github 超級架站術（MacOS, 2024）

### DISCUSSION THREAD

**Discussion by JohnnyXiao1024** [1101-GS4719-C/Python-Community-GS4719-C#1904] [General]
雖然我不是這學期的修課學生，但我猜想這裡或許會有人感興趣：

最近意外接觸到 Hexo 框架，支援 Markdown 語法真的深得我心
如果想架設一個部落格給自己使用，我相信 Hexo 會超適合你

網路上雖然有一些教學資源，但多半已經過時了
而且專為 MacOS 使用者所撰寫的安裝流程，更是少之又少

我基於自己的學習歷程，嘗試用更簡白的方式去進行介紹
目前只完成第一章，我會努力更新的，歡迎大家指教～
https://hackmd.io/@_jUPco9EQvmzW2lqVehtGQ/hexo

**Comment by benbebobo149** [comment]
我看了你的文章內容，覺得你好像對於DevOps蠻有興趣的，或許你可以學看看docker，把它包成image，這樣無論在哪個平台，只要一鍵就可以部署到你的主機上，就不會有你遇到的一些問題了。更進階的是可以結合GitHub的CICD，幫你檢查repo並自動部署到雲端上。我對你的發展前景充滿信心，加油！最後，感謝同學的分享，讓我們有機會彼此成長。
